packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -324,9 +352,17 @@ abstract class PrimaryPointerGestureRecognizer extends OneSequenceGestureRecogni
   @override
   void handleEvent(PointerEvent event) {
     assert(state != GestureRecognizerState.ready);
-    if (state == GestureRecognizerState.possible && event.pointer == primaryPointer) {
-      // TODO(abarth): Maybe factor the slop handling out into a separate class?
-      if (event is PointerMoveEvent && _getDistance(event) > kTouchSlop) {
+    if (event.pointer == primaryPointer) {
+      final bool isPreAcceptSlopPastTolerance =
+          state == GestureRecognizerState.possible
+          && preAcceptSlopTolerance != null
+          && _getDistance(event) > preAcceptSlopTolerance;
+      final bool isPostAcceptSlopPastTolerance =
+          state == GestureRecognizerState.accepted
+          && postAcceptSlopTolerance != null
+          && _getDistance(event) > postAcceptSlopTolerance;
```

Comment:
nit: the alignment here seems all off

Reply:
I'm assuming this is regarding https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#align-expressions. 

Done